app/main/controllers/privacy/RTMediaPrivacy.php

```diff
@@ -220,32 +267,44 @@ function select_privacy_ui( $echo = true, $element_id = false, $element_class =
 		}
 
 		if ( $echo ) {
-			echo $form->get_select( $attributes ); // @codingStandardsIgnoreLine
+			echo wp_kses( $form->get_select( $attributes ), wp_kses_allowed_html() );
```

Comment:
@vaishaliagola27 Make sure this won't break the output.

Reply:
Sure. Will check before merging.